netty/src/test/java/io/grpc/netty/NettyClientTransportTest.java

```diff
@@ -465,8 +466,10 @@ private NettyClientTransport newTransport(ProtocolNegotiator negotiator, int max
     }
     NettyClientTransport transport = new NettyClientTransport(
         address, NioSocketChannel.class, new HashMap<ChannelOption<?>, Object>(), group, negotiator,
-        DEFAULT_WINDOW_SIZE, maxMsgSize, maxHeaderListSize, keepAliveTimeNano, 1L, false, authority,
-        userAgent);
+        DEFAULT_WINDOW_SIZE, maxMsgSize, maxHeaderListSize,
+        KeepAliveManager.clampKeepAliveTimeInNanos(keepAliveTimeNano),
```

Comment:
if `enableKeepAlive` is true, the original nano values are too small. `keepAliveTimeout` being 1 nano definitely fails the test.